### DISCUSSION TITLE
Duplicate rule - which rule takes precedence ?

### DISCUSSION THREAD

**Discussion by VSevagen** [eslint/eslint#17015] [Q&A] [answered]
Hello,

I have an issue where there are duplicate rules in the .eslint.js
I would like to know which rule is taken into consideration when we have duplicates. 

My opinion is that it takes the last one defined since I'm assuming  the file is read from top to bottom.

Please let me know if I'm wrong.

Thanks

**Comment by Rec0iL99** [comment, marked as answer]
Hi @VSevagen, yes your assumption is correct.

```js
// .eslintrc.js
module.exports = {
  env: {
    commonjs: true,
    es2021: true,
    node: true,
  },
  overrides: [],
  parserOptions: {
    ecmaVersion: "latest",
  },
  rules: {
    "no-unused-vars": ["error"],
    "no-unused-vars": ["warn"],
  },
};
```

Rule configurations that are mentioned later in the `rules` object will override the ones that are mentioned first. In this case, the `no-unused-vars` rule will have the severity of `warn` finally in the resulting actual config. 

Thanks for using ESLint.

**Reply by VSevagen** [reply]
Thank you for replying :)
